lib/common/src/Amp.php

```diff
@@ -0,0 +1,261 @@
+<?php
+
+namespace Amp;
+
+use DOMElement;
+use DOMNode;
+
+/**
+ * Central helper functionality for all Amp-related PHP code.
+ *
+ * @package amp/common
+ */
+final class Amp
+{
+
+    /**
+     * List of Amp attribute tags that can be appended to the <html> element.
+     *
+     * @var string[]
+     */
+    const TAGS = ['amp', '⚡', '⚡4ads', 'amp4ads', '⚡4email', 'amp4email'];
```

Comment:
Elsewhere constants are defined on an `interface` but here they are on a `class`. Does that matter? Is the `interface` used for consistent API to access the constants?

Reply:
It does not really matter. I use interfaces if we only have contracts and classes if we also have behavior.

We could split this up, but coming up with meaningful names will be difficult then.